android/src/main/kotlin/com/airship/flutter/AirshipPlugin.kt

```diff
@@ -49,9 +56,35 @@ class InboxMessageViewFactory(private val registrar: Registrar) : PlatformViewFa
 
 class FlutterInboxMessageView(private var context: Context) : PlatformView, MethodCallHandler {
 
+    lateinit private var webviewResult:Result
+
     private val webView: UAWebView by lazy {
         val view = UAWebView(context)
-        view.webViewClient = UAWebViewClient()
+        view.webViewClient = object: UAWebViewClient() {
+            override fun onPageStarted(view: WebView, url: String?, favicon: Bitmap?) {
+                super.onPageStarted(view, url, favicon)
+                EventManager.shared.notifyEvent(WebviewLoadStartedEvent())
```

Comment:
I dont think you want to use `EventManager`,  instead these events should be sent through the `Registrar`/`Channel` in the `InboxMessageViewFactory`

Reply:
Ok, changed the approach